extensions/google-drive/src/utils.ts

```diff
@@ -161,6 +164,33 @@ ${new Date(file.createdAt).toLocaleString()}
 ${new Date(file.updatedAt).toLocaleString()}`;
 };
 
+export const fileStream = ({ stats = false }: { stats?: boolean } = {}) => {
+  const driveRootPath = getDriveRootPath();
+  const preferences = getPreferenceValues<Preferences>();
+
+  const filter = new Transform({
```

Comment:
```suggestion
  const transform = new Transform({
```

Reply:
Removed in favour of the `node-glob` function.